process/smartContract/process.go

```diff
@@ -290,6 +297,12 @@ func (sc *scProcessor) ExecuteSmartContractTransaction(
 		return nil
 	}
 
+	err = sc.txLogsProcessor.SaveLog(txHash, tx, vmOutput.Logs)
```

Comment:
maybe treat this as non-critical error ?

Reply:
I removed the return, left only debug message

Reply:
rename err to errNotCritical

Reply:
Renamed to `ignorableError` - since now we have three level of errors:
 - this one, ignorable error, which is only logged, but execution continues
 - `errNotCritical` which existed before, they interrupt execution, appear in SCR, but the return is nil
 - critical errors, such as the ones for `CalculateHash` or `SaveAccount`.

Is it ok?